Ascend.java

```diff
@@ -2,6 +2,15 @@
 // and prints them in increasing order.
 public class Ascend {
 	public static void main(String[] args) {
-		// Replace this comment with your code
-	}
+		int n = Integer.parseInt(args[0]);
+		int a = (int)(Math.random()*n);
+		int b = (int)(Math.random()*n);
+		int c = (int)(Math.random()*n);
+		int n1 = Math.min(a,Math.min(b,c));

```

Comment:
Lack of spacing between parameters of a function call:
 
int minValue = Math.min(input1, Math.min(input2, input3);